servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsServiceDiscovererBuilder.java

```diff
@@ -44,10 +49,44 @@
  */
 @Deprecated // FIXME: 0.43 - make package private
 public final class DefaultDnsServiceDiscovererBuilder implements DnsServiceDiscovererBuilder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultDnsServiceDiscovererBuilder.class);
+
+    /**
+     * @deprecated This system property is introduced temporarily as a way for users to skip binding and revert the
+     * preexisting behavior.
+     */
+    @Deprecated // FIXME: 0.43 - consider removing this system property
+    private static final String SKIP_BINDING_PROPERTY = "io.servicetalk.dns.discovery.netty.skipBinding";
+    private static final boolean SKIP_BINDING = getBoolean(SKIP_BINDING_PROPERTY);
```

Comment:
feel free to ignore: there seems to be little value in storing the value since it's only used to evaluate a static final field right below it.